src/AcronymList.js

```diff
@@ -32,6 +33,12 @@ const acronymCategories = (list = []) => {
 };
 
 const AcronymList = (props) => {
+  if (props.list && !props.list.length) {
+    return (
+      <NoResultsPage />
+    );
+  }
+
```

Comment:
Should we do the ```props.list``` check for the ```map``` method as well?

Reply:
Yeah maybe destructure list off with a default assignment

Reply:
👍 Updated.